src/LegacyPaymentGateways/Adapters/LegacyPaymentGatewayAdapter.php

```diff
@@ -123,11 +130,63 @@ public function handleBeforeGateway(array $legacyDonationData, PaymentGateway $r
                 $donation
             );
 
+            /**
+             * Add success, cancel and failed URLs to gateway data.  This will be used in both v2 and v3 forms so gateways can just refer to the gateway data.
+             *
+             * @unreleased
```

Comment:
Why are we adding a `@since` tag here? Wouldn't this go on the function itself?

Reply:
Some of the since tags are inline with the filter here but I can move them up